### PR TITLE
Create a new /info endpoint

### PR DESCRIPTION
**Pull request by pypingou**
This endpoint returns information about the number of sessions total and active associated
to this IP.

### CODE REVIEW

File: faitout/faitoutlib/__init__.py [hunk 1]
```diff
@@ -404,3 +404,21 @@ def get_stats(session):
...
+
+    :arg session: the session with which to connect to the faitout database.
+    :arg ipaddr: the IP address of interest.
+
+    """
+    output = {}
+
+    output['total_connections'] = model.Connection.search(
+        session, ip=ipaddr, cnt=True)
+    output['active_connections'] = model.Connection.by_ip(
```
Comment (jeremycline) [inline faitout/faitoutlib/__init__.py]: Why not use ``search`` for both queries or ``by_ip`` for both queries? They seem to do the same thing in this case.

**Comment by pypingou** [comment]
Good call, I just dropped by_id() and replaced it with calls to search() instead.

Thanks :)

File: faitout/templates/ip_stats.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+{% if stats['active_connections'] %}
+Active Connections:
+{% for connection in stats['active_connections'] %}
+  Connection #{{ loop.index }} created on {{ connection.creation_date.isoformat() }}
+{% endfor %}
+{% endif %}
+
+{% if ip in config['IP_UNLIMITED'] %}
+This IP can have un-limited sessions.
+{% elif ip in config['IP_UNLIMITED'] %}
```
Comment (jeremycline) [inline faitout/templates/ip_stats.html]: I think maybe this was meant to be 'IP_BLOCKED' or similar?

**Comment by jeremycline** [comment]
Other than the one kwarg which I think is wrong, this looks good 👍

File: faitout/templates/ip_stats.html (hunk 2 above)
Comment (pypingou) [inline faitout/templates/ip_stats.html]: That sounds right :D

**Comment by pypingou** [comment]
I'm going to merge this, thanks for the review :)

**Pull request merged**